index.js

```diff
@@ -117,4 +117,46 @@ app.post('/payments/create', async (request, result) => {
     }
 })
 
+// Endpoint used to edit customer
+app.post('/customers/edit', async (request, result) => {
+    const customerData = request.body;
+    console.log("Customer Edit Payload: ", customerData);
+
+    try {
+        const { name, tariffPlanId, siteId, meterNumber, hasmeter, isactive, id, customerUpdate } = customerData;
+        const { dateUpdated, customerIds, explanation, userId } = customerUpdate;
```

Comment:
Since there's only 1 user here, can we make `userId` just a string instead of a list of strings that will only contain 1 element (this prevents the need for indexing on line 140)?

We might have to change the Airtable schema so that only 1 user is responsible for a customer update

Reply:
done & changed main repo too